core/templates/services/playthrough-issues-backend-api.service.ts

```diff
@@ -35,7 +35,7 @@ import { UrlInterpolationService } from
 
 @Injectable({ providedIn: 'root' })
 export class PlaythroughIssuesBackendApiService {
-  private cachedIssues = null;
+  private cachedIssues: undefined | PlaythroughIssue[];
```

Comment:
Why can this be undefined ?

Reply:
changed the implementation slightly to remove `undefined` from the type